src/App.js

```diff
@@ -0,0 +1,23 @@
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import Greetings from './components/Greetings';
+import store from './redux/store';
+import './App.css';
+
+function App() {
+  return (
+    <Provider store={store}>
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route path="/greetings" element={<Greetings />} />
```

Comment:
- [ ] [Optional] You have done a great job. I just wanted to let you know that your app is functioning as expected. 
It would be better if you made only the greetings component and rendered the random greeting directly to the root route `/`.